.github/wiki/Test-The-Apply-Action-On-Development.md.md

```diff
@@ -0,0 +1,47 @@
+## Manually
```

Comment:
this file name has a redundant extension `.md.md`

Reply:
btw, I think the name of this wiki page isn't descriptive
> Test the apply action on development

What is it about? As the content focuses on generating a new phoenix project on the local development environment using the elixir template, should we change the name to reflex that?

Reply:
Fixed in bc86934 and 46d4171